core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
```

Comment:
can we implement `public static <T> T call(RpcCallable<T> callable)` 
by `public static <T> T call(Logger logger, RpcCallable<T> callable)`?